src/routes/user.js

```diff
@@ -717,4 +717,47 @@ router.get('/current', authenticate(), async (req, res) => {
   }
 });
 
+/**
+ * GET User Info by Username
+ * @param userName string
+ */
+router.get(
+  '/:name',
```

Comment:
Won't this clash with the `/:id` route? I think they should be something like `/id/1` and `/name/someone` to make sure there is no chance one handler will pre-empt the other.

Reply:
You are correct, if the username is numeric this route would never be reached.

Reply:
If it's alright with you, I'd rather make that change in the PR for user flavor notes and then I can include the frontend changes with an adjacent PR on that end